datatable/extras/aggregate.py

```diff
@@ -9,7 +9,8 @@
 from datatable.lib import core
 
 def aggregate(self, n_bins=500, nx_bins=50, ny_bins=50, max_dimensions=50, seed=0):
```

Comment:
Perhaps you should add some documentation here what this function does and what it returns. Because that' not trivial

Reply:
What is the best place to add it? 

I've been thinking this is a convenient way of defining this function. If one only needs the aggregated table it could be as simple as `aggregate(dt)`. If, in addition, one also needs the members it is `dt_members = aggregate(dt)`. If I do aggregation in-place, not sure if there are any more intuitive solutions. Any suggestions?

Reply:
I'm not saying the API is bad, just that it needs some form of documentation. See for example here: https://github.com/h2oai/datatable/blob/master/datatable/frame.py#L640-L653

Reply:
Sure, I got your point. Will add some documentation to this function. Thanks!